assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -238,6 +239,7 @@ const baseResolvers = {
 					const accountID = response.profiles[ 0 ].accountId;
 					dispatch( STORE_NAME ).receiveGetProfiles( response.profiles, { accountID, propertyID } );
 				}
+				/* eslint-enable sitekit/camelcase-acronyms */
```

Comment:
See above, let's annotate individually. There are some lines in here which should not have this rule disabled. It should only be disabled on lines where there is a violation which is acceptable (due to 3P code).